github-rtl-comments.user.js

```diff
@@ -55,9 +54,9 @@
 			toolbars = $$(".toolbar-commenting"),
 			indx = toolbars.length;
 		if (indx) {
-			button = document.createElement("button");
-			button.type = "button";
-			button.className = "ghu-rtl toolbar-item tooltipped tooltipped-n";
+			button = document.createElement("md-code");
+			button.type = "md-code";
```

Comment:
Thanks for your work, but `md-code` is specific to the code button. Please change these two lines back to `button`.